### DISCUSSION TITLE
If we can use Plot file to store data inside it?

### DISCUSSION THREAD

**Discussion by dungla2011** [Chia-Network/chia-blockchain#6680] [Ideas]
I wonder If we can to store data inside Plot files? 
For example, each plot file 108gb, from 1gb to 108gb, so I will store (replace) content from 2gb to 108gb (keep only first 1gb). So I will have 107gb to store my data? I mean we only keep header, meta data of plots.

Then HDD may be used in storage server on data center, do not waste many useless idle HDDs...

**So benefit: plot-user can store plot in Datacenter, and do not need keep HDD, Datacenter can share money with users. Both have benefit and don't waste anything ...**

It 's really green coin!
Any idea?

**Comment by dungla2011** [comment]
To now, Netspace > 20EiB = 2 million x 10TB HDD = about 200.000 server (each server has 10 disk x10TB)
So it's equivalent 200.000 server, may be bigger Google , Facebook ... and it's very fast bigger with increasing now.
It really wastes if We have not way to use space on HDDs

**Comment by cccat6** [comment]
No, and not possible. Chia is not looking for storage. If you have this idea, filecoin or swarm will be a better place for you.
The content in plots is not "trash". They are the key to find the proof.

**Reply by hashtafak** [reply]
Just not possible from this current version. Chia team can change their prototype to match network requirements (but not so fast). Or they can stay with something like, umm.. chia-classic 😎

**Reply by cccat6** [reply]
No, it will not possible in any modified Chia. If there is, it should at least 99% different from the Chia right now.
Don' think this meaningless thing. It just not possible. 0 chances. Even the modified version is not possible.
There is 0 possibility, and have no doubt that it will continue not possible in the long future.
